doc/userguide/rules/dcerpc-keywords.rst

```diff
@@ -0,0 +1,99 @@
+DCE/RPC
+=======
+.. role:: example-rule-emphasis
+
+Suricata supports decoding `DCE/RPC <https://en.wikipedia.org/wiki/DCE/RPC>`_
+directly in UDP or within SMB via the dcerpc application layer protocol. Because
+of this we are able to detect within DCE/RPC the interface UUIDs, operation
+numbers, and for bytes within the stub data. This is achieved by using the
+keywords ``dcerpc.iface`` and ``dcerpc.opnum``, and the sticky buffer
+``dcerpc.stub_data``.
+
+
+dcerpc.iface
+------------
+It is necessary for a DCE/RPC client to bind to a service before being able call
```

Comment:
being able to call it?

Reply:
Hmmm... maybe s/service/interface/?
"An interface is a set of remotely callable operations offered by a server and invokable by clients."
http://pubs.opengroup.org/onlinepubs/009629399/chap2.htm#tagcjh_05_01_01_01